refinery/core/utils.py

```diff
@@ -21,6 +21,9 @@
 import core
 import data_set_manager
 
+from core.search_indexes import DataSetIndex
```

Comment:
@hackdna I would love to, but neither `core` nor `data_set_manager` have a reference to `search_indexes`